app/components/common/TeamLeaderboardTable.component.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Image, Table, List} from 'semantic-ui-react';
+import { FormattedNumber } from 'react-intl';
+import './style.scss';
+
+const TeamLeaderboardTable = ({
+  data, prefix, image, dataKey, className, height, teams
```

Comment:
Do we need image and prefix props in this component?

Reply:
Should be resolved now